dataplicity/constants.py

```diff
@@ -3,11 +3,29 @@
 
 from os import environ
 
+
+def get_environ_int(name, default):
+    """Get an integer from the environment.
+
+    Args:
+        name (str): environment variable name
+        default (int): Default if env var doesn't exist or not an integer
+
+    Returns:
+        int: Integer value of env var.
+    """
+    try:
```

Comment:
you could also do:

value = environ.get(name, default)
if value.isdigit():
    return int(value)
return default

and you would save yourself from exception handling :D

Reply:
True, I don't necessarily dislike the exception though.